documentation/DotNetCli.md

```diff
@@ -13,7 +13,12 @@ For this add the following to the dependencies section of the project.json file:
       "type": "build"
     }
 ```
-
+For VS2017 / dotnet core 1.1.1 SDK 1.0.1 projects edit the `.csproj` file and mark the `StyleCop.Analyzers` as `PrivateAssets`:
+```xml
+    <ItemGroup>
+        <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta001" PrivateAssets="All" />
```

Comment:
`PrivateAssets` is the new way of specifying that a nuget dependency is used as a `build` asset:
https://marketplace.visualstudio.com/items?itemName=georgewfraser.vscode-javac